.travis.yml

```diff
@@ -106,6 +106,16 @@ matrix:
         - . $HOME/.cargo/env
         - cargo build --all-targets
 
+    # Fuchsia doesn't have external CI support, so we'll just make sure things compile
+    - language: generic
+      name: fuchsia
+      script:
+        - curl --proto '=https' --tlsv1.2 -sSf --output rustup.sh https://sh.rustup.rs
+        - sh rustup.sh -y --profile=minimal --default-toolchain 1.36.0 --target x86_64-fuchsia
+        - . $HOME/.cargo/env
+        - cargo build --all-targets
```

Comment:
You need to add `--target x86_64-fuchsia` to these two lines.  (you probably copied this from the Redox section, which makes the same mistake; I'll fix that separately).

Reply:
I don't think this is incorrect; the rustup invocation only installs the fuchsia target, so this does the right thing. The same is true for redox.

Reply:
It is incorrect.  I just checked in a fresh environment.  The rustup invocation installs both the native toolchain and the one that you request with `--target`.  So you need to do one of tho things:
* Use `--default-host` instead of `--target`, or
* Add `--target x86_64-fuchsia` to the `cargo build` line, as I suggested.